skrifa/src/outline/glyf/hint/engine/mod.rs

```diff
@@ -190,7 +190,7 @@ mod mock {
                 unscaled: vec![Default::default(); 32],
                 points: vec![Default::default(); 64],
                 point_flags: vec![Default::default(); 32],
-                contours: vec![32],
+                contours: vec![31],
```

Comment:
did somebody's tests uncover a bug?

Reply:
A bug in the test support code but that counts for something?

Reply:
half points